meta-openpower/recipes-phosphor/host/op-proc-control_git.bb

```diff
@@ -11,17 +12,26 @@ inherit autotools obmc-phosphor-utils pkgconfig pythonnative
 inherit systemd
 
 SRC_URI += "git://github.com/openbmc/openpower-proc-control"
-SRCREV = "b964c928156c2e71fe3bc9a2693b02cfbba5309c"
+SRCREV = "16ab00cb9383b17b8dd033a1cb300e2a013d55b1"
+SRC_URI += "git://github.com/ibm-openbmc/openpower-proc-control;branch=OP940"
```

Comment:
Should delete the other SRC_URI

Reply:
Hi Andrew, as per latest discussion @rfrandse is going to bump up rev